agents/vmware_soap/fence_vmware_soap.py

```diff
@@ -57,7 +57,8 @@ def soap_login(options):
 
 	try:
 		headers = {"Content-Type" : "text/xml;charset=UTF-8", "SOAPAction" : "vim25"}
-		conn = Client(url + "/vimService.wsdl", location=url, transport=RequestsTransport(verify=verify), headers=headers)
+		login_timeout = int(options["--login-timeout"])
```

Comment:
This doesnt work with --disable-timeout=true (enabled by default when run from Pacemaker 2.0+). Can you add ` or 60` after the `int()` here, so it at least gets a good chance to succeed for that use-case?

Reply:
Ok, a check for `--disable-timeout` has been added.